css-view-transitions-1/Overview.bs

```diff
@@ -476,27 +498,40 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	Animation type: discrete
 	</pre>
 
-	The 'view-transition-name' property "names" an element as participating in a view transition.
+	The 'view-transition-name' property “tags” an element
+	as potentially [=involved in a view transition=],
+	capturing it independently in the [=view transition tree=]
+	under the specified <dfn>view transition name</dfn>.
+	An element so captured is animated independently of the rest of the page.
 
 	<dl dfn-type=value dfn-for=view-transition-name>
 		: <dfn>none</dfn>
-		:: The [=/element=] will not participate in a view transition.
+		:: The [=/element=] will not participate independently in a view transition.
 
 		: <dfn><<custom-ident>></dfn>
-		:: The [=/element=] can participate in a view transition,
-			as either an old or new [=/element=],
-			with a <dfn dfn for>view transition name</dfn> equal to the <<custom-ident>>'s value.
+		:: The [=/element=] participates independently in a view transition--
+			as either an old or new [=/element=]--
+			with the specified [=view transition name=].
 
-			Note: The value <css>none</css> is invalid as a <<custom-ident>>.
+			The value <css>none</css> is excluded from <<custom-ident>> here.
+
+			Note: If this name is not unique
```

Comment:
It might be better to link to the relevant algorithm here. Something like:

```
Note: The presence of a `view-transition-name` doesn't guarantee that an element will participate independently in a view-transition. The property is ignored if an element skips its contents, has more than one box fragment, or is not rendered.
Also, if this name is not unique ...
See capture the old state/capture the new state algorithm for the precise conditions.
```

Reply:
Added a cross-reference, good idea.